programs/eosio-blocklog/CMakeLists.txt

```diff
@@ -1,14 +1,22 @@
 add_executable( eosio-blocklog main.cpp )
 
+find_package(progressbar 2.0 QUIET)
```

Comment:
Adding an (albeit optional) dependency for a progress bar seems.. a bit wasteful. I can't even figure out what library provides this.. it looks like _maybe_ this https://github.com/doches/progressbar but it's lacking  `progressbar_new_percent_with_format_and_tumbler` and `progressbar_update_percent`.

And doing a google or github search for `progressbar_new_percent_with_format_and_tumbler` turns up completely empty and searching for ubuntu packages with "progressbar" files in them likewise comes up empty.

Is anyone even going to have this library? Is this better suited as a submodule or copy/pasta job instead? (the one linked above adds a dependency on to ncurses, which is something we'll want to avoid as a hard requirement though)

Reply:
It's my version, which I haven't pushed to Github yet.  And it's just for me, because I'll be running the thing 261 times on Mainnet and wanted to see what it's doing.

Reply:
It's a public library, now found [here](https://github.com/jgiszczak/progressbar).  It builds cleanly and silently without it, only using it if it's present and detected by CMake.

Reply:
We need to be judicious on the number of dependencies we pick up and this doesn't seem like a worthwhile dependency addition. It only adds trivial functionality "just for @jgiszczak" via a niche package that realistically nobody will ever have nor will be tested.  I also would like to avoid hard dependencies on ncurses so bundling the code as-is isn't an option either.